### PR TITLE
[EDU-3136] feat: add activate ddos

### PR DESCRIPTION
**Pull request by hannah-sk**
### Changes

* Added new page for Activate DDoS for Secure menu.

[EDU-3136]: https://aziontech.atlassian.net/browse/EDU-3136?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+meta_tags: secure, ddos, unmetered
+namespace: docs_guides_secure_advanced_activate_ddos
+permalink: /documentation/products/guides/secure/activate-ddos/
+menu_namespace: secureMenu
+---
+
+Once you activate [DDoS Protection Unmetered](/en/documentation/products/edge-firewall/ddos-protection/), you'll be protected against DDoS attacks. The protections works with unmetered mitigation and it automatically mitigates attacks against your domains and applications.
+
+:::note
+DDos Protection Unmetered is availabe for clients with **Business**, **Enterprise**, and **Mission Critical** Service Plans.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:13]: Maybe add link to service plans?

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: How to activate DDoS Protection Unmetered
+description: Activate the DDoS Protection Unmetered module to protect your applications agains volumetric attacks.
+meta_tags: secure, ddos, unmetered
+namespace: docs_guides_secure_advanced_activate_ddos
+permalink: /documentation/products/guides/secure/activate-ddos/
+menu_namespace: secureMenu
+---
+
+Once you activate [DDoS Protection Unmetered](/en/documentation/products/edge-firewall/ddos-protection/), you'll be protected against DDoS attacks. The protections works with unmetered mitigation and it automatically mitigates attacks against your domains and applications.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:10]:
```suggestion
Once you activate [DDoS Protection Unmetered](/en/documentation/products/edge-firewall/ddos-protection/), you'll be protected against DDoS attacks. The protection works with unmetered mitigation and it automatically defends your domains and applications against attacks.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+You can activate this module at any point of your journey. It'll be helpful to protect your applications against volumetrics attacks as it has the always-on intelligence to mitigate complex attacks.
+
+It doesn't require any configurations, only that you activate it in your account and in a firewall:
+
+1. Access Real-Time Manager (RTM) > [select the Account menu](/en/documentation/products/guides/how-to-access-rtm/#managing-the-account).
+2. Select **Billing & Subscriptions** > **Subscriptions**.
+3. In the **Edge Firewall** section, find **DDoS Protection Unmetered** and enable the switch.
+
+Next, go to the [edge firewall](/en/documentation/products/guides/secure/configure-main-settings#modules) in which you want to activate DDoS Protection Unmetered:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:24]:
```suggestion
Next, go to [Edge Firewall](/en/documentation/products/guides/secure/configure-main-settings#modules) and proceeds as follow:
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Select an edge firewall.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:26]:
```suggestion
1. Select the edge firewall in which you want to activate DDoS Protection Unmetered.
```

**Review by MarianaAguilera — APPROVED** [review]
left a few suggestions.

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+2. Enable the **DDoS Protection Unmetered** switch.
+
+Now, the domains associated with the specific edge firewall will be protected against DDoS attacks.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:29]:
```suggestion
Now the domains associated with the specific edge firewall will be protected against DDoS attacks.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+You can activate the DDoS module in any edge firewall you desire, as well as use it along with other modules.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:31]:
```suggestion
You can activate the DDoS module in any edge firewall, as well as use it along with other modules.
```

**Review by bru-andrade — APPROVED** [review]
😎 belíssimo

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx (hunk 1 above)
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:13]:
```suggestion
DDoS Protection Unmetered is availabe for clients with **Business**, **Enterprise**, and **Mission Critical** Service Plans.
```

Comment (gabriel-azion) [reply src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:13]:
```suggestion
DDoS Protection Unmetered is available for clients with **Business**, **Enterprise**, and **Mission Critical** Service Plans.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+menu_namespace: secureMenu
+---
+
+Once you activate [DDoS Protection Unmetered](/en/documentation/products/edge-firewall/ddos-protection/), you'll be protected against DDoS attacks. The protections works with unmetered mitigation and it automatically mitigates attacks against your domains and applications.
+
+:::note
+DDos Protection Unmetered is availabe for clients with **Business**, **Enterprise**, and **Mission Critical** Service Plans.
+:::
+
+You can activate this module at any point of your journey. It'll be helpful to protect your applications against volumetrics attacks as it has the always-on intelligence to mitigate complex attacks.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:16]:
```suggestion
You can activate this module at any point in your journey. It'll be helpful to protect your applications against volumetric attacks as it has the always-on intelligence to mitigate complex attacks.
```

File: src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx (hunk 2 above)
Comment (hannah-sk) [inline src/content/docs/en/pages/secure-journey/advanced-configurations/activate-ddos.mdx:10]:
```suggestion
Once you activate [DDoS Protection Unmetered](/en/documentation/products/edge-firewall/ddos-protection/), you'll be protected against DDoS attacks. The protection works with unmetered mitigation and it automatically mitigates attacks against your domains and applications.
```

**Pull request merged**